File: cfgs/java_cfgs_foo/PROGRAM_BINARY_DECIMAL_CONVERSION_1.java
Nodes
1;special;;;
2;statement;String num = foo_1;14;
3;statement;int dec_value = 0;15;
4;statement;int base = 1;16;
5;statement;int len = fun_1();17;
6;statement;int i = foo_1 - 1;18;
7;conditional;foo_1 >= 0;19;
8;conditional;fun_1(foo_2) == '1';21;
9;statement;dec_value += foo_1;21;
10;statement;base = foo_1 * 2;22;
11;statement;i--;20;
12;statement;return foo_1;24;
13;exit;;;
Edges
0;2;
1;3;
2;4;
3;5;
4;6;
5;7;
6;8;12;
7;9;10;
8;10;
9;11;
10;7;
11;13;
12;

File: cfgs/pythoncfgs/PROGRAM_BINARY_DECIMAL_CONVERSION_1.py
Nodes
1;statement;foo_1 = foo_2;8;
2;statement;foo_1 = 0;9;
3;statement;foo_1 = 1;10;
4;statement;foo_1 = fun_1(foo_2);11;
5;conditional;fun_1(foo_1 - 1, -1, -1);12;
6;statement;return foo_1;16;
7;conditional;foo_1[foo_2] == '1';13;
8;statement;foo_1 = foo_2 * 2;15;
9;statement;foo_1 += foo_2;14;
10;exit;;
Edges
0;2;
1;3;
2;4;
3;5;
4;6;7;
5;10;
6;8;9;
7;5;
8;8;
9;